### DISCUSSION TITLE
fetchmail.pl results in Pg.c: loadable library and perl binaries are mismatched error

### DISCUSSION THREAD

**Discussion by gotkube** [postfixadmin/postfixadmin#768] [General]
Hoping someone can help me out. I recently updated my Slackware system to -current, which included upgrades to various packages, including Perl. I had postfixadmin running on my system prior to the update and had no problems. Since upgrading and trying to fetch email with fetchmail.pl in the ADDITIONS directory, I'm getting this error:

`Pg.c: loadable library and perl binaries are mismatched (got first handshake key 0xf180080, needed 0xf980080)`

I tried rebuilding my PostgreSQL server/libraries; I had similar issues with PHP & a few other packages I build from source on my system, so I assumed this was similar, but apparently not.

Can someone help shed some light on this and how to resolve it? I'd appreciate the help :)

**Comment by gotkube** [comment]
LOL sorry nvm. Turns out I had to rebuild my DBD::Pg module for Perl.
`> cpan DBD::Pg`
...did the trick for me in-case anyone has this or a similar problem again

**Reply by DavidGoodwin** [reply]
I was hoping it was a packaging/slackware problem :)
